src/test/rustdoc-gui/run-on-hover.goml

```diff
@@ -0,0 +1,4 @@
+goto: file://|DOC_PATH|/test_docs/fn.foo.html
```

Comment:
Please add a comment explaining what this test is about.

Reply:
Done!